AmazonKinesisVideoDemoApp/README.md

```diff
@@ -1,54 +1,14 @@
 # Running AmazonKinesisVideoStreaming Sample
 
-More information: [https://docs.aws.amazon.com/kinesisvideostreams/latest/dg/producer-sdk-android.html](https://docs.aws.amazon.com/kinesisvideostreams/latest/dg/producer-sdk-android.html)
+## 1. Provision and setup
 
-## 1. Create a user pool
-  * Go to https://console.aws.amazon.com/cognito/
-  * Click `Manage your User Pools`
-  * Click `Create a user pool`
-  * Fill-in `Pool name`
-  * Click `Review defaults`
-  * Click `Create user pool`
-  * Copy `Pool Id` :clipboard:
-  * Select `App clients` in the left nav.
-  * Click `Add an app client`
-  * Fill-in `App client name`
-  * Click `Create app client`
-  * Click `Show details` and copy `App client id` and `App client secret` :clipboard:
-    * ![Shows show details button](screenshots/click_show_details.png) `-->` ![](screenshots/copy_app_client_id_and_secret.png)
+Follow the prerequisites section to provision AWS resources required to run this sample, and assign the appropriate permissions the IAM role: [https://docs.aws.amazon.com/kinesisvideostreams/latest/dg/producer-sdk-android.html](https://docs.aws.amazon.com/kinesisvideostreams/latest/dg/producer-sdk-android.html)
```

Comment:
also maybe link directly to the prerequisites section?  https://docs.aws.amazon.com/kinesisvideostreams/latest/dg/producersdk-android-prerequisites.html

Reply:
Done